third_party/curl.BUILD

```diff
@@ -0,0 +1,37 @@
+# Builds CURL from a distribution.
+
+load("@io_opentelemetry_cpp//bazel:curl.bzl", "CURL_COPTS")
+
+package(features = ["no_copts_tokenization"])
+
+config_setting(
+    name = "windows",
+    values = {"cpu": "x64_windows"},
+    visibility = ["//visibility:private"],
+)
+
+config_setting(
+    name = "osx",
+    values = {"cpu": "darwin"},
+    visibility = ["//visibility:private"],
+)
+
+cc_library(
+    name = "curl",
+    srcs = glob([
+        "lib/**/*.c",
```

Comment:
`*.cc`?

Reply:
CURL seems to use all `.c` files, see: https://github.com/curl/curl/tree/master/src